Cargo.toml

```diff
@@ -11,7 +11,7 @@ edition = "2021"
 repository = "https://github.com/near/near-api-rs"
 description = "Rust library to interact with NEAR Protocol via RPC API"
 
-exclude = ["resources", "examples", "tests"]
```

Comment:
Do you think it should be part of the package?

Reply:
this thing requires `examples` to be included to published sources, otherwise it looks like doc won't be able to be built
https://github.com/near/near-api-rs/blob/5d49851641fbe1b7781a55ed2c5928087d594ae3/src/contract.rs?plain=1#L276

Reply:
examples are 560 lines in total compared to 7923 lines (everything, with examples), so i hope it won't be a taxing addition. And the proportion of examples code <> lib code will probably stay the same